app/src/utils/user-name.ts

```diff
@@ -1,7 +1,7 @@
 import { i18n } from '@/lang';
 import { User } from '@directus/types';
 
-export function userName(user?: Partial<User>): string {
+export function userName(user?: Pick<User, 'first_name' | 'last_name' | 'email'> | null): string {
```

Comment:
We should really correct the `User` type. I have stumbled across this before, the properties of a user can be `null` but should still (all?) be present. Maybe this would allows us to simply use `user?: User` here?

Reply:
It's also because in most cases, the user is fetched in different ways. So that shared User type isn't applicable in a lot of those cases 🤔

Reply:
Yeah... For this case however `Partial<User>` is fine because the function isn't expecting specific attributes of `User`. Updated accordingly.